k8s.gcr.io/images/k8s-staging-kubebuilder/OWNERS

```diff
@@ -0,0 +1,10 @@
+# See the OWNERS file documentation:
+#  https://github.com/kubernetes/community/blob/master/contributors/devel/owners.md
+
+approvers:
+- pwittrock
+- camilamacedo86
+- varshaprasad96
```

Comment:
We need add here @vincepri and @sbueringer 

This one will probably only used for we build and promote : https://github.com/kubernetes-sigs/kubebuilder-release-tools/issues/60 so far